process_report/util.py

```diff
@@ -0,0 +1,30 @@
+import datetime
+import json
+
+
+def get_institution_from_pi(institute_map, pi_uname):
+    institution_key = pi_uname.split("@")[-1]
+    institution_name = institute_map.get(institution_key, "")
+
+    if institution_name == "":
+        print(f"Warning: PI name {pi_uname} does not match any institution!")
+
+    return institution_name
+
+
+def load_institute_map() -> dict:
+    with open("process_report/institute_map.json", "r") as f:
+        institute_map = json.load(f)
+
+    return institute_map
+
+
+def get_iso8601_time():
+    return datetime.datetime.now().strftime("%Y%m%dT%H%M%SZ")
```

Comment:
Why not just use the `datetime.datetime.isoformat` method?

```
>>> datetime.datetime.now().isoformat(timespec='seconds')
'2024-05-24T13:16:40'
```

You get delimiters in the date, but otherwise the format is identical.

```suggestion
def get_iso8601_time():
    return datetime.datetime.now().isoformat(timespec='seconds')
```